<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelControlWidgetItem</class>
 <widget class="QWidget" name="ModelControlWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModelControlWidgetItem</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="VisibilityButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Visible</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ImplantDesign.qrc">
         <normaloff>:/ImplantDesign/Resources/AnnotationVisibility.png</normaloff>:/ImplantDesign/Resources/AnnotationVisibility.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CategoryButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
       </property>
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="opacSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Opacity</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ColorPickerButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Color</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
       </property>
       <property name="displayColorName" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed{background-color: qconicalgradient(cx:1, cy:0, angle:176.3, stop:0.202247 rgba(255, 127, 148, 255), stop:1 rgba(255, 255, 255, 255));}

QPushButton{border:2px solid gray;border-radius:10px;padding:2px 4px;}







</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ImplantDesign.qrc">
         <normaloff>:/ImplantDesign/Resources/AnnotationDelete.png</normaloff>:/ImplantDesign/Resources/AnnotationDelete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Save</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:pressed{background-color: qconicalgradient(cx:1, cy:0, angle:176.3, stop:0.202247 rgba(255, 127, 148, 255), stop:1 rgba(255, 255, 255, 255));}

QPushButton{border:2px solid gray;border-radius:10px;padding:2px 4px;}







</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ImplantDesign.qrc">
         <normaloff>:/ImplantDesign/Resources/DownloadExtension.png</normaloff>:/ImplantDesign/Resources/DownloadExtension.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImplantDesign.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>VisibilityButton</sender>
   <signal>clicked()</signal>
   <receiver>ModelControlWidgetItem</receiver>
   <slot>OnVisibilityOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnVisibilityOff()</slot>
 </slots>
</ui>
